src/subcommand/convert_main.cpp

```diff
@@ -469,6 +485,7 @@ void help_convert(char** argv) {
          << "    -r, --in-rgfa-rank N   import rgfa tags with rank <= N as paths [default=0]" << endl
          << "    -b, --gbwt-in FILE     input graph is a GBWTGraph using the GBWT in FILE" << endl
          << "        --ref-sample STR   change haplotypes for this sample to reference paths (may repeat)" << endl
+         << "        --hap-sample STR   change generic paths for this sample to haplotype paths (may repeat)" << endl
```

Comment:
I don't think this (or the docs for the modified functions) captures what's really happening. What we're doing is taking generic paths and changing them to haplotype paths, using the generic path's name (in "locus" currently) as the new sample name, and making up a new locus name.

The wording for the option makes it sound like the user is specifying a sample name, but there's no such thing as "generic paths for this sample" because generic paths don't belong to samples. Really the user is specifying a locus/full generic path name, which gets transformed into a sample name in the output.

The option here is sort of structured as if it would do the opposite of `--ref-sample`, but it operates on generic paths instead, so it probably should be structured and described more distinctly.